index.js

```diff
@@ -395,22 +432,17 @@ module.exports = {
     "no-restricted-syntax": "off",
     "no-tabs": "error",
     "no-ternary": "off",
+    "no-trailing-spaces": "error",
 
     // Disabled because leading underscores are used to mark private members in
     // classes and trailing underscores are used to avoid collisions with
     // reserved words.
     "no-underscore-dangle": "off",
     "no-unneeded-ternary": "error",
     "no-whitespace-before-property": "error",
-    "object-curly-newline": "off",
-    "object-curly-spacing": ["error", "always"],
+    "object-curly-newline": "error",
```

Comment:
I think this only had one or two things to fix when I changed it.

Reply:
Before this rule eslint have a [nonblock-statement-body-position](https://eslint.org/docs/rules/nonblock-statement-body-position).